simplegrid/cow.py

```diff
@@ -61,7 +63,10 @@ def learn(self, reward, done):
 
     def draw(self, display):
         display.circle(
-            self.x, self.y, math.sqrt(min(0.64, 2 * self.energy / MAX_ENERGY)), self.__class__.COLOR
+            self.x,
+            self.y,
+            math.sqrt(min(0.64, 2 * self.energy / MAX_ENERGY)),
+            self.__class__.COLOR,
```

Comment:
It feels like there is a line length change here in the config that is not part of this PR

Reply:
I set line length to 120. I think you do too.
But somehow in master it's 99.
I will try to change it again.